codegen/type_converter.go

```diff
@@ -142,6 +162,9 @@ func (c *TypeConverter) genConverterForStruct(
 	fromPrefix string,
 	indent string,
 	fieldMap map[string]FieldMapperEntry,
+	requestType string,
```

Comment:
What is `requestType` ? It seems like it goes on the function name.

Can we store the `requestType` parameter on the `TypeConverter` class itself ?